src/services/axios.ts

```diff
@@ -1,7 +1,12 @@
 import axios from 'axios'
 
-const baseURL = 'https://lunes.tuerantuer.org/api'
-// const baseURL = 'https://lunes-test.tuerantuer.org/api'
+let baseURL: string
```

Comment:
If you wanted to get rid of the let inline if would work as well :D

Reply:
yeah you are right. Will change this and be a little ashamed :D